truckersmp-cli

```diff
@@ -557,17 +617,17 @@ def update_game():
     +app_update {}{} validate
     +quit""".format(
       steamcmd, args.account, args.gamedir, args.steamid,
-      " -beta {}".format(args.beta) if args.beta else ""))
+      " -beta {}".format(branch) if branch else ""))
```

Comment:
Once `--beta temporary_1_36` is specified, even if `--beta` is *not specified next time*, `steamcmd` won't upgrade to 1.37. After specifying `--beta [invalid_branch_name]`, `steamcmd` upgraded to 1.37.

```suggestion
      " -beta {}".format(branch if branch else "__latest__")))
```

The specified branch name (`temporary_1_36`) is stored in `$XDG_DATA_HOME/truckersmp-cli/[Game_name]/data/steamapps/appmanifest_[AppId].acf`:
```
        "UserConfig"
        {
                "betakey"               "temporary_1_36"
        }
```

But removing this file changed nothing.

Do you know better solution?

Reply:
Sorry, the "invalid" branch name can be set after `get_beta_branch_name()`:
```python
    else:
        game = "ats" if args.ats else "ets2"
        beta_branch_name = get_beta_branch_name(game)
        branch = beta_branch_name if beta_branch_name else "__latest__"
```

```suggestion
      " -beta {}".format(branch)))
```

Reply:
I actually forgot about this strange behavior.

> Do you know better solution?

`public` is the normal branch name and the thing we want here.